### ISSUE TITLE
[BUG] `Module has no attribute "client"`

### ISSUE THREAD

**Issue by maxcountryman** [vemel/mypy_boto3_builder#20]
After updating to `boto3-stubs` version `1.12.48` (this matches the `boto3` version I'm using) I continue to see this error:

```
Module has no attribute "client"
```

A simple test case that consistently reproduces the problem:

```python
import boto3

sts_client = boto3.client("sts")
```

**Comment by vemel** [comment]
Hmm, looks like you did not generate index files with

```bash
mypy_boto3
```

Let me know if it helps.

**Comment by maxcountryman** [comment]
> Hmm, looks like you did not generate index files with
> 
> ```shell
> mypy_boto3
> ```
> 
> Let me know if it helps.

I did try that, but continue to see the same error.

**Comment by vemel** [comment]
Cannot reproduce locally. I will take a closer look today. Thank you for the report.

**Comment by maxcountryman** [comment]
If there’s any additional context I can provide to help track this down, I’m more than happy to.

**Comment by vemel** [comment]
I can only recommend to uninstall `boto3-stubs` and index files then install again

```bash
python -m mypy_boto3 --clean
python -m pip uninstall -y boto3-stubs
python -m pip freeze | grep mypy-boto3 | xargs python -m pip uninstall -y

python -m pip install `boto3-stubs[sts]`
```

Let me know if it helps

**Comment by maxcountryman** [comment]
So far we haven’t had any luck. I think I did read somewhere that having `boto` as a project dependency might cause issues with this package? I think `moto` might be requiring it even tho we use `boto3`.

**Comment by vemel** [comment]
Hi @maxcountryman ! I have just released builder 2.0. It makes index build (dynamic type annotations) step optional. If you just install `boto3-stubs[sts]`

```python
import boto3
from mypy_boto3_sts import STSClient

sts_client = boto3.client("sts") # type: botocore.client.BaseClient
sts_client: STSClient = boto3.client("sts") # type: STSClient
```

If you build index with `mypy_boto3` command, then first case will also have a correct type. However, you might want to provide type explicitly.

Closing this issue, but let me know if it works for you.
